specs/experimental/fault-proof/stage-one/fault-dispute-game.md

```diff
@@ -459,3 +476,12 @@ Each move bisects the historical state of L2 and eventually, `MAX_GAME_DEPTH` is
 can be settled conclusively. Dishonest players are disincentivized to participate, via backwards induction,
 as an invalid claim won't remain uncontested. Further incentives can be added to the game by requiring
 claims to be bonded, while rewarding game winners using the bonds of dishonest claims.
+
+### Finalization
+
+Once the game is resolved, if the claim is shown to be valid, the FDG reports its state to the
+Anchor State Registry. The Anchor State Registry verifies that the request to update an anchor
+state comes from a FDG contract created by the Dispute Game Factory contract, confirms that the
+game resolved in favor of the defender, and confirms that the updated state would be newer than
+the current anchor state. If these conditions are true, the Anchor State Registry updates the
+anchor state for the given game type.
```

Comment:
oh and it's now tracking per game type instead of just the single respected game type.

Reply:
Yes, that's correct. I chatted with @clabby and we decided that this was cleaner and avoided adding extra functionality to the `OptimismPortal` contract since it's considered a critical contract.

Reply:
This seems like a sane approach. It could be a clear way to communicate with a list of invariants for state modifications to the contract + pseudocode. It could also be nice to include an interface